pfrl/experiments/evaluator.py

```diff
@@ -82,10 +83,17 @@ def _run_episodes(
 
 def _hrl_run_episodes(
     env, agent: HIROAgent, n_steps, n_episodes, max_episode_len=None, logger=None,
+    step_number=None, video_outdir=None
 ):
     """Run multiple episodes and return returns."""
     assert (n_steps is None) != (n_episodes is None)
 
+
+    if step_number is not None:
```

Comment:
I think you should first assign to an boolean

```
record_video = step_number is not None

if record_video: 
...
```

Reply:
or we can rename `step_number` to `step_number_for_recording` or something that has `record` component. 
I prefer the former method though.

Reply:
I actually found this feature in `VideoRecorder` called `enabled` variable.

```
        enabled (bool): Whether to actually record video, or just no-op (for convenience)
```

We can turn this on and off whenever `step_number==None`

Reply:
ah great, thx!

Reply:
fixed.